models/reviews.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection")
+
+exports.fetchReviewById = id => {
```

Comment:
Just a reminder, be careful when not wrapping arguments in parenthesis. This works as there is only a single argument, but if there were more than 1, no parentheses would be invalid syntax :)